plugin-server/src/types.ts

```diff
@@ -666,17 +671,6 @@ export interface ClickHouseEvent extends BaseEvent {
 /** Event in a database-agnostic shape, AKA an ingestion event.
  * This is what should be passed around most of the time in the plugin server.
  */
```

Comment:
👍  like what you did here ... less types is better :) And the `PreIngestionEvent` indeed matches to `PostHogEvent` almost, extended the comment, so they don't diverge too much in the future.
```suggestion
 * Keep easy to convert to/from PostHogEvent (in plugin-scaffold repo) that's used in plugins.
 */
```